languages/python/oso/oso/exceptions.py

```diff
@@ -4,9 +4,9 @@ class AuthorizationError(Exception):
 
 class NotFoundError(AuthorizationError):
     """
-    Thrown by the ``authorize`` method of an ``Enforcer``. This error indicates
-    that the actor is not only not allowed to perform the given action, but also
-    is not allowed to ``"read"`` the given resource.
+    Thrown by the ``authorize`` method of an ``Oso`` instance. This error
+    indicates that the actor is not only not allowed to perform the given
+    action, but also is not allowed to ``"read"`` the given resource.
 
     Most of the time, your app should handle this error by returning a 404 HTTP
     error to the client.
```

Comment:
Couple references to `Enforcer` below this line (GitHub doesn't allow me to leave this comment on the exact line)